index.js

```diff
@@ -129,6 +132,29 @@ module.exports.makeApp = function () {
     next()
   })
 
+  /*
+   Route to check the status of the application by
+   calling the package_search of backend with row count as 0
+   to check whether the application is returning response or not
+ */
+  app.use('/status', async (req, res) => {
+    const url = resolve(
+      config.get('API_URL'),
+      `package_search?rows=0`
```

Comment:
Not sure if `package_search` is the right CKAN API to call here.

Reply:
OK, let's keep `package_search` as we want to check if db and solr services are responding.